src/components/molecules/Cv/components/BasicData.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { Box } from '../../../quarks';
+import { Page } from '../../../atoms';
+import { ICvTypes } from '../Cv.types';
+
+export const BasicData: FC<IBasicDataProps> = ({
+  studentStatus = '',
+  workingHours = '',
+  sanitationBook = '',
+  drivingLicences = [],
+}) => {
+  return (
+    <Box fontScale={2}>
+      <Page.Heading>Informacje podstawowe:</Page.Heading>
+      {studentStatus && (
```

Comment:
Jeśli "ifujemy" wszystkie elementy to teoretycznie może być sytuacja, w której wyrenderujemy tylko `heading`. Może warto dodać całemu komponentowi warunek? Dunno.